web/src/user/LibraryPage/ApplicationEmptyState.ts

```diff
@@ -50,7 +50,7 @@ export class LibraryPageApplicationEmptyList extends AKElement {
                 <a
                     aria-disabled="false"
                     class="cta pf-c-button pf-m-secondary"
-                    href="/if/admin/${href}"
+                    href="${globalAK().api.base}if/admin/${href}"
```

Comment:
This repetition starts to feel like it needs to be, oh, I dunno, a _function_ somewhere.  :-)